pkg/controller/storageprofile-controller.go

```diff
@@ -44,8 +54,24 @@ func (r *StorageProfileReconciler) Reconcile(_ context.Context, req reconcile.Re
 	if err := r.client.Get(context.TODO(), req.NamespacedName, storageClass); err != nil {
 		return reconcile.Result{}, err
 	}
+	if _, err := r.reconcileStorageProfile(storageClass); err != nil {
+		return reconcile.Result{}, err
+	}
+
+	// Alert for incomplete profiles
+	numIncomplete := 0
+	storageProfileList := &cdiv1.StorageProfileList{}
+	if err := r.client.List(context.TODO(), storageProfileList); err != nil {
```

Comment:
looks like a good way for a generic metric, I suppose `List` is already cached, is it?

Reply:
Yes, this was my main concern, I am definitely using the cached client (as opposed to unCachedClient), so I think this is not too expensive